package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "datastruct",
+  "version": "0.1.0",
+  "description": "re impelmenting push pop shift unshift",
+  "main": "index.js",
+  "scripts": {
+    "test": "gulp watch-mocha"
```

Comment:
Given your gulpfile, this should probably just be `gulp`, as `gulp watch-mocha` doesn't run tests